javascript/javascript1/week3/app.js

```diff
@@ -0,0 +1,178 @@
+// Item array removal
+const names = [
+  "Peter",
+  "Ahmad",
+  "Yana",
+  "kristina",
+  "Rasmus",
+  "Samuel",
+  "katrine",
+  "Tala",
+];
+const nameToRemove = "Ahmad";
+names.splice(1, 1);
+
+console.log(names);
+
+
+// When will we be there??
+const travelInformation = {
+  speed: 50,
+  destinationDistance: 432,
+};
+
+function travelTime() {
+  const time =
+    (travelInformation.destinationDistance / travelInformation.speed) * 60;
+  const hours = Math.floor(time / 60);
+  const minutes = Math.floor(time % 60);
+  return `${hours} hours and ${minutes} minutes`;
+}
+console.log(travelTime());
+
+
+// Series duration of my life
+const seriesDurations = [
+  {
+    title: "Lost",
+    days: 9,
+    hours: 3,
+    minutes: 0,
+  },
+  {
+    title: "Game of thrones",
+    days: 3,
+    hours: 1,
+    minutes: 0,
+  },
+  {
+    title: "Sopranos",
+    days: 3,
+    hours: 14,
+    minutes: 0,
+  },
+  {
+    title: "The Wire",
+    days: 2,
+    hours: 12,
+    minutes: 0,
+  },
+];
+
+let totalTime = [];
+
+function timeOfLife() {
+  for (let item of seriesDurations) {
+    const MovieName = item.title;
+    const daysToYear = item.days / 365;
+    const hoursToYear = item.hours / 8766;
+    const minutesToYear = item.minutes / 525600;
+    const movieTime = (
+      (daysToYear + hoursToYear + minutesToYear) *
+      1.25
+    ).toFixed(4);
+    const newMovieTime = (daysToYear + hoursToYear + minutesToYear) * 1.25;
+    console.log(`${MovieName} took ${movieTime}% of my life`);
+    totalTime.push(newMovieTime);
+  }
+}
+
+timeOfLife();
+
+const initialValue = 0;
+const totalTimeSpent = totalTime.reduce(
+  (accumulator, currentValue) => accumulator + currentValue,
+  initialValue
+);
```

Comment:
Very nice usage of reduce. You can actually write 0 directly, instead of storing it in a variable.